README.md

```diff
@@ -38,6 +38,18 @@ trayer --edge bottom \
 # forget about i3bar (｢・ω・)｢
 ```
 
+# Components Configuration
+
+## github-alert
+
+This is a tool that remind you to **commit** to github everyday :P
+
+* Get a Personal Access Token from [here](https://github.com/settings/tokens/new)
+* tick repo, user and notifications
+* then you will get a 40 characters access_token
+* copy the file called config.json.sample to .config.json
```

Comment:
Why `.config.json` instead of `config.json` ?

It is nice to keep a top-level config file. Since it's top-level, I prefer it to be more generalized than for one specific component. e.g.

- define fields for each component.
- use module name (the name used by `require`) as the field name.

```json
{
  "github_alert" : {
    "token": "...",
    "user": "..."
  },
  "another_component": {}
}
```